test/integration/029_docs_generate_tests/test_docs_generate.py

```diff
@@ -43,6 +44,25 @@ def run_and_generate(self, extra=None):
         self.assertEqual(len(self.run_dbt()), 1)
         self.run_dbt(['docs', 'generate'])
 
+    def assertRecent(self, timestr):
+        """Given a timestring in '%Y-%m-%dT%H:%M:%SZ' format (ISO8601), assert
+        that it represents a time before now and a time after 24h ago.
+
+        We can't just set the time via freezegun.freeze_time because that
+        breaks SSL, and a lot of these tests use SSL.
+        """
+        now = datetime.utcnow()
+        yesterday = now + timedelta(days=-1)
+        parsed = datetime.strptime(timestr, '%Y-%m-%dT%H:%M:%S.%fZ')
+        self.assertLess(
+            yesterday, parsed,
+            'parsed date {} happened over 24h ago'.format(parsed)
+        )
+        self.assertGreater(
+            now, parsed,
+            'parsed date {} happened in the future'.format(parsed)
+        )
```

Comment:
hah interesting note about freezegun.freeze_time. i think this is fine as a single-use hack. let's make sure not to overuse this.